### PR TITLE
:sparkles: page not found

### PR DESCRIPTION
**Pull request by ntchou**

### CODE REVIEW

**Comment by HugoGresse** [comment]
fuck I've done it too  :(

File: public/locales/fr/not-found.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+    "page-title": "Page introuvable",
+    "page-content": "Désolé nous n'avons pas pu trouver cette page.",
```
Comment (Abibibi) [inline public/locales/fr/not-found.json:3]: Please add a comma after "Désolé"

File: src/components/App/index.js [hunk 2]
```diff
@@ -56,6 +57,7 @@ const App = () => {
             <Route path={`/${t('url.blog')}`} exact component={Blog} />
             <Route path={`/${t('url.blog')}/:post`} exact component={Blog} />
             <Route path={`/${t('url.credits')}`} exact component={Credits} />
+            <Route component={NotFound} />
```
Comment (Abibibi) [inline src/components/App/index.js:60]: `NotFound` component appears underneath each page of the website. Maybe using a `Switch` component would solve the problem

instantVisio-notFound

File: public/locales/fr/not-found.json (hunk 1 above)
Comment (ntchou) [reply public/locales/fr/not-found.json]: done

File: src/components/App/index.js (hunk 2 above)
Comment (ntchou) [reply src/components/App/index.js]: You're right `Switch` was missing

File: src/pages/NotFound/index.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import { useTranslation } from 'react-i18next'
+import DefaultLayout from '../../layout/Default'
+import { Link } from 'react-router-dom'
+
+export default function NotFound() {
+    const { t } = useTranslation('not-found')
+
+    return (
+        <DefaultLayout title={`${t('page-title')} - Instant Visio`}>
```
Comment (HugoGresse) [inline src/pages/NotFound/index.js:10]: Maybe you could get some styles from https://github.com/Instant-Visio/InstantVisio-WebApp/pull/158/files#diff-0bcbe3f27cb0b8b5e1803367f395ae4c?

**Comment by Abibibi** [comment]
> fuck I've done it too :(

@HugoGresse :/

To avoid this in the future, let's all make sure that an owner is assigned for each issue created.

**Pull request merged**